gulpfile.js

```diff
@@ -25,32 +26,38 @@ gulp.task('prettify:verify', ['prettify:sources:verify', 'prettify:jsx:verify'])
 
 var prettify_options = {
   html: {
-    eol: '\n',
-    indentSize: 2
+    indent_size: 2
   },
   css: {
-    eol: '\n',
```

Comment:
We can let these eol go, as this is the default value already. So this doesn't change anything.